source/fundamentals/auth.txt

```diff
@@ -72,7 +73,7 @@ mechanisms depending on what MongoDB versions your server supports:
      - MongoDB 3.0, 3.2, 3.4, and 3.6
 
    * - ``MONGODB-CR``
-     - MongoDB 3.6 and earlier
+     - MongoDB 3.6 and earlier if the server doesn't support``SCRAM-SHA-1``
```

Comment:
This still isn't correct. This table is for the *default* it looks like. [MongoDB 3.0](https://docs.mongodb.com/v3.0/core/security-mongodb-cr/) changed the default from `MongoDB-CR` to `SCRAM-*`.